app/styles/app.css

```diff
@@ -1178,6 +1178,8 @@ input#search:-ms-input-placeholder {
 body * { visibility: hidden; }
 #printcontent * { visibility: visible; }
 #printcontent { position: absolute; top: 100px; left: 100px; width: 160%;height: 100%;}
+    #printreceiptcontent * { visibility: visible; }
+    #printreceiptcontent { position: absolute; top: 100px; left: 100px; width: 160%;height: 100%;}
```

Comment:
i meant conerting this id selector http://www.w3schools.com/cssref/sel_id.asp into a class and reusing the same, rather than introduce a style for each id which would lead to lot of duplication